public/js/index.js

```diff
@@ -594,6 +594,7 @@ function initWikiForm() {
                         function (data) {
                             preview.innerHTML = '<div class="markdown">' + data + '</div>';
                             emojify.run($('.editor-preview')[0]);
+                            $('.editor-preview').autolink();
```

Comment:
@andreynering I've experimented hard with it for a long time. Often rendering on backend resulted in broken children links or image links. So instead it is now done on frontend, with a better, conservative yet effective implementation, which operates the DOM, not the wall of text. No links or image urls are now broken, as now it respects tags and their children. It is quite fast, and it offloads servers.